### DISCUSSION TITLE
What is bin/dufs being used for

### DISCUSSION THREAD

**Discussion by johnfoxii** [gtxaspec/wz_mini_hacks#763] [Q&A]
i see dufs in the bin directory. 
it shows up as a Gen:NN.Mirai.36804 in VT
other versions of packaged (dufs) code are questionable

VirusTotal checksums
d111f9182a4428a58f77439198b561eaab1bb12949fbfa30c91b96a10e37224e
e73f71594eef7067353b84850c35ac0d55b246fc2f2a1fdb86264d579b1b796c

**Comment by gtxaspec** [comment]
dufs was added in:
https://github.com/gtxaspec/wz_mini_hacks/pull/705

**Comment by gtxaspec** [comment]
I ran it through virus total:
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/12115272/2c269a76-2a4f-4e60-ad1b-0ea57640cc07)
and only 1 hit, using `BitDefenderTheta`.  

It could be a false positive, but to err on the safe side, I'll remove the existing version of the binary and recompile a fresh version from the dufs repo https://github.com/sigoden/dufs
